### PR TITLE
dbeaver/pro#1530 remove eclipse.ui dependency from draw2d & transfer theming to gef

### PR DESCRIPTION
**Pull request by Destrolaric**
Closes dbeaver/pro#1530
@azoitl Transferred theming to the GEF plugin. It will replace basic functionality upon plugin activation. If only draw2d is present old colors will be used unfortunately

### CODE REVIEW

**Comment by Destrolaric** [comment]
`NEEDRESTART-VER: 3.5
NEEDRESTART-KCUR: 5.15.0-1035-azure
`
I think there are issues present with the CI/CD.

**Comment by azoitl** [comment]
@Destrolaric thx for the fix. I had a first look at the output of the windows and linux builds. The tests are back to where we were before the color management improvement. Will give it some tries on my machine tonight or tomorrow. I also have some changes for the failing tests that I would like to try with your PR.

File: org.eclipse.draw2d/src/org/eclipse/draw2d/BasicColorProvider.java [hunk 1]
```diff
@@ -0,0 +1,128 @@
+/*******************************************************************************
+ * Copyright (c) 2000, 2023 IBM Corporation and others.
```
Comment (azoitl) [inline org.eclipse.draw2d/src/org/eclipse/draw2d/BasicColorProvider.java:2]: As this is a sole contribution by you this line should be:
    2023 DBeaver Corporation

File: org.eclipse.draw2d/src/org/eclipse/draw2d/BasicColorProvider.java [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *     Georgii Gvinepadze - georgii.gvinepadze@dbeaver.com
+ *     Serge Rider - serge@dbeaver.com
```
Comment (azoitl) [inline org.eclipse.draw2d/src/org/eclipse/draw2d/BasicColorProvider.java:11]: In the other eclipse projects we do for first files:

  Georgii Gvinepadz, Serge Rider - initial API and implementation

File: org.eclipse.gef/src/org/eclipse/gef/ThemeConstants.java [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2022 IBM Corporation and others.
+ * Copyright (c) 2000, 2023 IBM Corporation and others.
```
Comment (azoitl) [inline org.eclipse.gef/src/org/eclipse/gef/ThemeConstants.java:2]: similar then above this copyright header should not feature IBM

File: org.eclipse.gef/src/org/eclipse/gef/GEFColorProvider.java [hunk 4]
```diff
@@ -1,32 +1,35 @@
...
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *     IBM Corporation - initial API and implementation
- *     Georgii Gvinepadze - georgii.gvinepadze@dbeaver.com
- *     Serge Rider - serge@dbeaver.com
- *******************************************************************************/
-package org.eclipse.draw2d;
+* Copyright (c) 2000, 2023 IBM Corporation and others.
```
Comment (azoitl) [inline org.eclipse.gef/src/org/eclipse/gef/GEFColorProvider.java:4]: similar then above this copyright header shouldn't feature IBM. Sorry that I missed that the first time.

**Review by azoitl — CHANGES_REQUESTED** [review]

**Comment by azoitl** [comment]
@Destrolaric while your commit fixed the problems with Draw2d and Zest I now see the GEF tests failing because of an issue in the InternalGEFPlugin. I haven't seen this before therefore I'm somehow thinking this could come from your commit. Do you have any idea what is going on here?

**Comment by Destrolaric** [comment]
@azoitl Initialization of the provider is done from the plugin. I am currently thinking about what causes it to fail.

**Comment by Destrolaric** [comment]
@azoitl Hmm, are this two tests have failed previously?

**Comment by azoitl** [comment]
> @azoitl Hmm, is these two tests failed previously?

I think so. We had several failing test before you started working on GEF Classic. I would look later into them.

**Review by azoitl — APPROVED** [review]
thx for the updates they look good.

**Pull request merged**